src/ComunicaUpdateEngine.js

```diff
@@ -0,0 +1,47 @@
+import ComunicaEngine from 'ldflex-comunica';
+import auth from 'solid-auth-client';
+
+/**
+ * An extension of ComunicaEngine that delegates
+ * SPARQL UPDATE queries directly to the documents
+ * using authenticated request.
+ */
+export default class ComunicaUpdateEngine extends ComunicaEngine {
```

Comment:
Maybe we can just extend `ComunicaEngine`?
This type of inheritance, which adds functionality instead of defining or modifying, is not really common.

Reply:
Not sure if I understand. Do you want to move this logic into ldflex-comunica instead? This would not really work because of the `solid-auth-client` dependency.

Reply:
No you're right, this is fine.